### DISCUSSION TITLE
Icon selection is text input, not a drop down like other icon fields

### DISCUSSION THREAD

**Discussion by KevBro360** [totaldebug/atomic-calendar-revive#1245] [Support]
Screenshot attached.  The icon field is not a typical drop down or type / search like other icon selections in HA.  It is only a text input field - is this how it's supposed to be?

![Screenshot 2023-10-20 004212](https://github.com/totaldebug/atomic-calendar-revive/assets/143299121/de2328d5-fb6f-4045-ba32-4705e0013b6f)

**Comment by marksie1988** [comment]
It is meant to be like this. As per the docs https://docs.totaldebug.uk/atomic-calendar-revive you type in the MDI icon that you want. 

Creating a drop-down for this would be a nightmare to maintain. 

I'll have a look and see if I can replicate how they do it in HA where when you type it starts to show the available icons as I think that is done via the MDI icon library
